src/components/CollectionPagesSection.jsx

```diff
@@ -210,7 +210,7 @@ const CollectionPagesSection = ({ collectionName, pages, siteName, isResource, r
                                     itemIndex={pageIdx}
                                     category={collectionName}
                                     siteName={siteName}
-                                    fileName={page.fileName || page.name} // temporary fix
+                                    fileName={page.fileName} // temporary fix
```

Comment:
nit: can remove comment now

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/412/commits/1ce7153a42365a2ccb6faa68be0fa264fb721d53